### DISCUSSION TITLE
Bed Temp

### DISCUSSION THREAD

**Discussion by Dakweed666** [mainsail-crew/mainsail#1893] [Help]
I slice using Orcaslicer and have my bed temp set to 60 for the first layer and 55 for all the rest of the print, mainsail is ignoring the 55 setting and doing the whole print at 60, any ideas?

**Comment by meteyou** [comment]
Mainsail is only a Webinterface and has nothing to do with the Gcode.

Pls upload an example Gcode.

**Reply by Dakweed666** [reply]
[Gear.zip](https://github.com/mainsail-crew/mainsail/files/15450031/Gear.zip)

**Comment by meteyou** [comment]
the temp change is in your gcode, so klipper should execute it. do you have any macros for M140? pls upload also your klippy.log.

![image](https://github.com/mainsail-crew/mainsail/assets/8167632/919ada1c-37e2-4cae-bf09-34224b313edc)
